core/contracts/financial-templates/implementation/WETH9.sol

```diff
@@ -63,17 +63,17 @@ contract WETH9 {
         return address(this).balance;
     }
 
-    function approve(address guy, uint wad) public returns (bool) {
+    function approve(address guy, uint256 wad) public returns (bool) {
```

Comment:
Who came up with these horrible variable names?

Reply:
what? you dont want to `wad` your `guy`? is you `flip` not `flop`ed yet ya `shut?`

Reply:
^^ Maker 🐳